### PR TITLE
Update docs.md

### PR DESCRIPTION
**Pull request by mkozjak**

### CODE REVIEW

File: 04.Artifacts/08.Update-Modules/docs.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ Then add execute permission to the script:
 chmod +x web-file
 ```
 
-Your Mender client is now will be able to handle any Artifact containing a `web-file` Update Module Artifact.
+Your Mender client will now be able to handle any Artifact containing a `web-file` Update Module Artifact.
```
Comment (lluiscampos) [inline 04.Artifacts/08.Update-Modules/docs.md]: I think the original intention was to say "Your Mender client **is** now able to..."

**Comment by lluiscampos** [comment]
Please rebase your work to have a single change set. Thanks

**Comment by mkozjak** [comment]
Done

**Comment by lluiscampos** [comment]
Great! Merging.

Thank you for your contribution.

**Pull request merged**